spool.go

```diff
@@ -60,6 +61,8 @@ func (p *SpoolPage) Layout(gtx layout.Context) layout.Dimensions {
 					func() layout.FlexChild {
 						if isConnected {
 							return layout.Rigid(button(th, p.connect, connectIcon).Layout)
+						} else if isConnecting {
+							return layout.Rigid(button(th, p.connect, connectIcons[connectIconIdx]).Layout)
 						}
 						return layout.Rigid(button(th, p.connect, disconnectIcon).Layout)
 					}(),
```

Comment:
what's the point of using an anonymous function call here instead of a switch-case on the line before `return layout.Flex{Axis..` ?

Reply:
oh man this pr is messed up. sorry. i pushed some different code and none of this exists any more. sorry.